tests/stress/stress_test.py

```diff
@@ -303,7 +311,7 @@ def main():
 
     if args.heavy:
         print('Setting the heavy configuration...')
-        args.large_file_size_gb = 10
+        args.large_file_size_gb = 32
```

Comment:
could set this to `min(10, gigabytes of ram on the machine)`

Reply:
I'm not sure if it's easy to get the memory of the machine programmatically since this script runs remotely.